specification/apimanagement/resource-manager/readme.md

```diff
@@ -28,15 +28,56 @@ These are the global settings for the ApiManagement API.
 title: ApiManagementClient
 description: ApiManagement Client
 openapi-type: arm
-tag: package-preview-2019-12
+tag: package-2019-12
 ```
 
 
+### Tag: package-2019-12
+
+These settings apply only when `--tag=package-2019-12` is specified on the command line.
+
+```yaml $(tag) == 'package-2019-12'
```

Comment:
Add a space before `yaml`.

Reply:
@raych1 This entire PR was generated by Open api hub. I didn’t push any commit. Maybe we need to fix something there. I can fix it here.